pyproject.toml

```diff
@@ -1,8 +1,41 @@
 [build-system]
-# AVOID CHANGING REQUIRES: IT WILL BE UPDATED BY PYSCAFFOLD!
-requires = ["setuptools>=46.1.0", "setuptools_scm[toml]>=5", "wheel"]
+requires      = ["setuptools>=61.0.0", "wheel"]
 build-backend = "setuptools.build_meta"
 
-[tool.setuptools_scm]
-# See configuration details in https://github.com/pypa/setuptools_scm
-version_scheme = "no-guess-dev"
+[project]
+name = "actinia-api"
+version = "3.3.1"
```

Comment:
Is it possible to change the version automatacally in a workflow?

Reply:
In general it would be possible, although this workflow would need to run before the PyPI release workflow, but both after a release. Also here - I would prefer to do this in a separate MR. Until then, I added a RELEASE.md which indicates to update the version before a release.
If you agree, I will create a ticket for that.

Reply:
yes. please create a ticket for this

Reply:
Issue created https://github.com/mundialis/actinia-api/issues/8